### PR TITLE
Implement sqlalchemy to connect to postgres -- epr #4

### PR DESCRIPTION
**Pull request by Nkuruba**
@epr #4

### CODE REVIEW

**Comment by Nkuruba** [comment]
> Please see rocketchat re. pulling latest skeleton branch, linting, and variable naming.

Pulled latest changes and updated code as per lint.

File: api/.env.example [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # This is a sample .env file that can be used for local development.
-ORIGINS="http://localhost http://localhost:3000 http://127.0.0.1 http://127.0.0.1:3000"
\ No newline at end of file
+ORIGINS="http://localhost http://localhost:3000 http://127.0.0.1 http://127.0.0.1:3000"
+POSTGRES_HOST="localhost"
+POSTGRES_USER="epr"
+POSTGRES_PASSWORD="epr"
+POSTGRES_DATABASE="epr_db"
```
Comment (Sybrand) [inline api/.env.example:6]: Can we just call the database epr? Having a database called db seems redundant.
```suggestion
POSTGRES_DATABASE="epr"
```

File: api/orm/models/employee.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, String, Integer
```
Comment (Sybrand) [inline api/orm/models/employee.py:1]: Can we please rather have a models.py with the employee model in it (see https://www.python.org/dev/peps/pep-0020/ - "Flat is better than nested".

File: api/orm/models/employee.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+from sqlalchemy import Sequence
+from db_init import BASE
+
+
+class Employee(BASE):
+    __tablename__ = 'EMPLOYEE'
+
+    id = Column(Integer, Sequence('employee_id_seq'), primary_key=True, nullable=False)
+    idir = Column(String, nullable=False)
```
Comment (Sybrand) [inline api/orm/models/employee.py:10]: This has to be a unique key

File: api/orm/db_init.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+""" Setup database to perform CRUD transactions
+"""
```
Comment (Sybrand) [inline api/orm/db_init.py:2]: What did you model the file structure after? I'm looking at https://fastapi.tiangolo.com/tutorial/sql-databases/ and see they're using "database.py", "models.py" etc. any reason for deviating from that?

File: api/orm/models/employee.py (hunk 3 above)
Comment (Nkuruba) [reply api/orm/models/employee.py:10]: added unique constraint.

File: api/orm/models/employee.py (hunk 2 above)
Comment (Nkuruba) [reply api/orm/models/employee.py:1]: Would re-structure as per EPR #19

File: api/orm/db_init.py (hunk 4 above)
Comment (Nkuruba) [reply api/orm/db_init.py:2]: Would re-structure as per EPR #19

File: api/.env.example (hunk 1 above)
Comment (Nkuruba) [reply api/.env.example:6]: Updated it.

**Pull request merged**